usersync/syncer.go

```diff
@@ -113,30 +113,30 @@ func resolveDefaultSyncType(syncerConfig config.Syncer) SyncType {
 // macro substitution regex
 var (
 	macroRegexExternalHost = regexp.MustCompile(`{{\s*\.ExternalURL\s*}}`)
-	macroRegexSyncerKey    = regexp.MustCompile(`{{\s*\.SyncerKey\s*}}`)
+	macroRegexBidderName   = regexp.MustCompile(`{{\s*\.BidderName\s*}}`)
```

Comment:
This is a breaking change and can cause hosts trouble. Please keep `SyncerKey` as you add `BidderName`, where both resolve to the bidder name.

Reply:
Done